### ISSUE TITLE
OIDC jmap.linagora.com: always asked to relogin

### ISSUE THREAD

**Issue by chibenwa** [linagora/tmail-flutter#2835]
### Description

Since the switch to OIDC for jmap.linagora.com

My twake mail application was logged out 5 time in a short time frame and ulimately I have 5 refresh tokens on lemon side:

```
Sun Apr 26 09:13:28 2026
Sun Apr 26 09:51:42 2026
Sun Apr 26 09:59:38 2026
Sun Apr 26 10:20:07 2026
Sun Apr 26 10:43:51 2026
```

Expectation: 

 - Understand what happens
 - Not ask me to relogin when I have a refresh token valid.
 
 Cc @yadd

**Comment by guimard** [comment]
I can confirm the bug and also confirm that previous `refresh_token` are still available in Lemon database

**Comment by hoangdat** [comment]
related https://github.com/linagora/tmail-flutter/pull/2751

**Comment by hoangdat** [comment]
fixed in `v0.12.0`
- frequency refresh token, make a pressure to write on Hive database (single thread db of Flutter)
- should close `Hive::box` when also want to open it in background thread
